plugin.go

```diff
@@ -78,6 +82,21 @@ func (p *Plugin) Exec(client *storage.Client) error {
 
 	p.bucket = client.Bucket(strings.Trim(bname, "/"))
 
+	// If in download mode, call the Download method
+	if p.Config.Download {
+		bname, remainingPath := extractBucketName(p.Config.Source)
+		p.Config.Source = remainingPath
+
+		p.bucket = client.Bucket(strings.Trim(bname, "/"))
+
+		log.Println("Downloading objects ...")
```

Comment:
Nit: also log the bucket and path in this log - like `Downloading objects from bucket: %s using path: %s`

Reply:
Added log statement